media_driver/agnostic/gen11/renderhal/renderhal_g11.cpp

```diff
@@ -721,6 +721,8 @@ MOS_STATUS XRenderHal_Interface_g11::IsOvrdNeeded(
     pAttriVe    = (PMOS_CMD_BUF_ATTRI_VE)(pCmdBuffer->Attributes.pAttriVe);
     pGenericPrologParamsG11 = dynamic_cast<PRENDERHAL_GENERIC_PROLOG_PARAMS_G11>(pGenericPrologParams);
 
+    if (pAttriVe)
+    {
```

Comment:
something is wrong with alignment here. tabs? please, fix.

Reply:
Indent was missed in the brackets. Fixed. Thanks!